presto-main/src/main/java/io/prestosql/execution/QueryStateTimer.java

```diff
@@ -170,6 +170,12 @@ private void endAnalysis(long now)
         analysisTime.compareAndSet(null, nanosSince(beginAnalysisNanos, now));
     }
 
+    /**
+     * Records the last time that Presto act on the query, such as retrieve the result
```

Comment:
I didn't understand the use of "such as retrieve the result".

Did you use this to exemplify an act?
If so, maybe we could this on the description and not on the summary.

Reply:
improved